README.md

```diff
@@ -257,6 +255,9 @@ Conduct](https://github.com/common-workflow-language/common-workflow-language/bl
 * Ward Vandewege, Curoverse Inc. / Arvados; https://orcid.org/0000-0002-2527-6949
 * Alexander Wait Zaranek, Curoverse Inc. / Arvados; https://orcid.org/0000-0002-0415-9655
 
+
+<a name="Leadership_Team"></a>
```

Comment:
Anchor are auto-created: http://www.commonwl.org/#CWL_Leadership_Team

Reply:
Nevermind, this is for constant anchor naming between GitHub and commonwl.org

Reply:
Yes, I used the HTML version at first, but realized GitHub likes - instead of _. Could do absolute URI though.